redhat-cop/openshift-management
# Repo maintenance and new github action for prune-ocp-projects

#### What is this PR About?
Doing some clean-up due to multiple PRs introducing similar functionality but in different areas of the repo. This PR consolidates and restructures the content a bit to be consistent. 

It also introduces github actions for buliding the `prune-ocp-projects` image automatically. 

#### How do we test this?
N/A - check github actions for no errors. 

Related PR to add `prune-ocp-project` to quay.io: https://github.com/redhat-cop/org/pull/207

cc: @redhat-cop/casl
